Draw a fuller arrow

bobvanderlinden opened the pull request:
The current arrow is composed of 3 thin lines. It would be nice to have a more fatter arrow, so that the individual lines of the arrow aren't as prominent.

The arrow looked like:

![image](https://github.com/user-attachments/assets/45f667fd-d9d6-4cd4-bed3-45d9b94bfc41)

(from small to large, unfilled first, then filled)

Now they look like:

![image](https://github.com/user-attachments/assets/9798bf90-db0f-4e03-b327-40dd9eef9855)

gabm commented:
I like the looks imho, code looks good to me! Would be a nice addition.

Currently I see these things

- fixing the bug you discovered
- can you post a before -after image?
- doc: 
  - looking this PR
  - adding an ASCII drawing as glossary (nice to have)

gabm commented:
I like the looks imho, code looks good to me! Would be a nice addition.

Currently I see these things

- fixing the bug you discovered
- can you post a before -after image?
- doc: 
  - looking this PR
  - adding an ASCII drawing as glossary (nice to have)

bobvanderlinden commented:
I've updated the PR:

* Updated the description with before and after screenshots.
* Added better description in the comments with an attempt at ASCII-art representation :sweat_smile:
* Made sure the styles are being used by the arrow drawing. Filled style is the new fat arrow. The unfilled style uses the separate lines approach. The sizes for the two styles are being reused.

This does change the sizes of the unfilled style a bit compared to what was used previously, but it is consistent with filled style this way. Let me know what you think!

gabm commented:
I like the result, looks good to me! 

If u feel that it's ready, please mark as ready and I'll merge.

bobvanderlinden commented:
Awesome 😄 Sorry, forgot this was still in draft 😅 It's ready now!
